index.html

```diff
@@ -1,12 +1,337 @@
+                    <li>
+                        <a href="#"><img src="Media/twitter.svg" /> </a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Media/Linkedin icon.svg" /></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Media/mediu.svg" /></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Media/github.svg" /></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Media/angellist.svg" /></a>
+                    </li>
+                </ul>
+                <button class="last-btn m-b" type="button">Get My Resume</button>
+            </div>
+            <div class="languages">
+                <ul class="languages-div language">
+                    <li>Languages</li>
+                    <i class="fa fa-angle-down" aria-hidden="true"></i>
+                </ul>
+                <div class="wrapper">
+                    <div class="languages-div skills">
+                        <img src="Media/Ellipse 1.png" />
+                        <h5 class="m-l">javascript</h5>
+                    </div>
+                    <div class="languages-div skills">
+                        <img src="Media/Ellipse 1 (1).png" />
+                        <h5 class="m-l">CSS</h5>
+                    </div>
+                    <div class="languages-div skills">
+                        <img src="Media/Ellipse 1 (2).png" />
+                        <h5 class="m-l">HTML</h5>
+                    </div>
+                </div>
+                <ul class="languages-div framework">
+                    <li>Framework</li>
+                    <i class="fa fa-angle-right" aria-hidden="true"></i>
+                </ul>
+                <ul class="languages-div framework">
+                    <li>Skills</li>
+                    <i class="fa fa-angle-right" aria-hidden="true"></i>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <footer class="section-4 ">
+        <div class="section-4-container">
+            <h2 class="contact">Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project
+                that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/xyyaowqk" method="post">
+                <ul>
+                    <li>
+                        <label for="name">
+                            <input class="full-name" type="text" placeholder="Lorem ipsum" id="name" maxlength="30"
+                                required name="fullname">
+                        </label>
+                    </li>
+                    <li>
+                        <label for="email">
+                            <input type="email" placeholder="loremipsum@gmail.com" id="email" maxlength="30" required
```

Comment:
[]Optional]Perhaphs changing the email placeholder `placeholder="loremipsum@gmail.com" ` would make it more appealing and easy to understand to the user.Something more user friendly like "youremail@gmail.com" Would look more appealing.